### DISCUSSION TITLE
GitHub火狐代理出错

### DISCUSSION THREAD

**Discussion by jerry460** [BeyondDimension/SteamTools#1393] [Q&A] [answered]
当对火狐进行代理时，github会出现证书错误，停止代理并点击安装证书后再打开依旧报错
如果先打开一次GitHub再开始加速则可以正常打开
如果把所有GitHub网页关闭后再打开github会出现证书错误
steam++为最新版，平台为win10

**Comment by AigioL** [comment, marked as answer]
https://steampp.net/faq#CommunityAccelerate
